neo/Network/P2P/Payloads/TransactionAttribute.cs

```diff
@@ -10,6 +10,17 @@ public class TransactionAttribute : ISerializable
         public TransactionAttributeUsage Usage;
         public byte[] Data;
 
+        public CosignerUsage DataAsCosignerUsage()
```

Comment:
```c#
Data.AsSerializable<CosignerUsage>();
```

Reply:
thanks Erik, this was really ugly and I couldnt think on something better :joy: